src/components/shared/propTypes.js

```diff
@@ -0,0 +1,8 @@
+import propTypes from '@dhis2/prop-types'
+
+export const toggleGroupOptionsProp = propTypes.arrayOf(
+    propTypes.shape({
+        label: propTypes.string.isRequired,
+        value: propTypes.string.isRequired,
+    })
+).isRequired
```

Comment:
I think, although shared between all components, we should move the `isRequired` to the components. It increases re-usability of this prop type and makes it more clear in the components' prop type definitions